spec/webauthn/authenticator_attestation_response_spec.rb

```diff
@@ -65,26 +65,17 @@
 
   describe "origin validation" do
     let(:original_origin) { "http://localhost" }
-    let(:challenge) { SecureRandom.random_bytes(16) }
-    let(:original_challenge) { WebAuthn::Utils.ua_encode(challenge) }
+    let(:challenge) { fake_challenge }
+    let(:encoded_challenge) { WebAuthn::Utils.ua_encode(challenge) }
     let(:client_data_json) {
-      hash_to_encoded_json(challenge: authenticator_encode(challenge),
-                           clientExtensions: {},
-                           hashAlgorithm: "SHA-256",
-                           origin: origin,
-                           type: "webauthn.create")
-    }
-    let(:auth_data) {
-      # TODO Move this to a spec helper which receives options so
-      # that it can be configured for different test cases
-      [73, 150, 13, 229, 136, 14, 140, 104, 116, 52, 23, 15, 100,
-       118, 96, 91, 143, 228, 174, 185, 162, 134, 50, 199, 153,
-       92, 243, 186, 131, 29, 151, 99, 1, 0, 0, 0, 0].pack('c*')
+      encoded_fake_client_data_json(challenge: challenge,
+                                    origin: origin)
     }
+
     let(:attestation_object) {
       hash_to_encoded_cbor(fmt: "none",
                            attStmt: {},
-                           authData: auth_data)
+                           authData: fake_authenticator_data)
```

Comment:
Should we use `encoded_fake_attestation_object` here?

Reply:
Yeap 👍 Discovered that other helper latter in the refactor. Good catch :)